extras/vagrantExample/src/main/vagrant/Vagrantfile

```diff
@@ -200,74 +247,93 @@ EOF
+		# Right now it's on dropbox, but eventually it'll be on maven...
+		echo "Downloading quietly, this will take some minutes with no output..."
+		download --output ${ryaIndexing}.zip https://dl.dropbox.com/s/7e74yiuq4jmu0od/${ryaIndexing}.zip?raw=1  || exit 112
+	fi
+	sudo mkdir --parents ${ryaIndexing}
+	sudo unzip -q -o ${ryaIndexing}.zip -d ${ryaIndexing}
 	
+    # before continueing, wait for tomcat to deploy wars:
+    waitForDeploy /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/
+    waitForDeploy /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/
+
 	# soft linking the files doesn't seem to work in tomcat, so we copy them instead :(
-	sudo cp rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution/dist/lib/* /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/
-	sudo cp rya.indexing.example-${RYA_EXAMPLE_VERSION}-distribution/dist/lib/* /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/
+	sudo cp ${ryaIndexing}/dist/lib/* /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/ || exit 113
+	sudo cp ${ryaIndexing}/dist/lib/* /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/    || exit 114
 
 	# These are older libs that breaks tomcat 7
-	sudo rm /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/servlet-api-2.5.jar
-	sudo rm /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/jsp-api-2.1.jar
-	sudo rm /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/servlet-api-2.5.jar
-	sudo rm /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/jsp-api-2.1.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/servlet-api-2.5.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/jsp-api-2.1.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/servlet-api-2.5.jar
+	sudo rm --force /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/jsp-api-2.1.jar
 	
 	sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/webapps/openrdf-workbench/WEB-INF/lib/
 	sudo chown -R tomcat7:tomcat7 /var/lib/tomcat7/webapps/openrdf-sesame/WEB-INF/lib/
 
-	# Download and install new templates for OpenRdf WorkBench
-	download -L -O https://www.dropbox.com/s/dgw63m66nubyy4z/rya.vagrant.example-${RYA_EXAMPLE_VERSION}.jar
-	sudo mkdir rya.vagrant.example-${RYA_EXAMPLE_VERSION}
-	sudo unzip rya.vagrant.example-${RYA_EXAMPLE_VERSION}.jar -d rya.vagrant.example-${RYA_EXAMPLE_VERSION}
-	sudo cp rya.vagrant.example-${RYA_EXAMPLE_VERSION}/*.xsl /var/lib/tomcat7/webapps/openrdf-workbench/transformations/
+	echo "Downloading and installing new templates for OpenRdf WorkBench"
+	ryaVagrant=rya.vagrant.example-${RYA_EXAMPLE_VERSION}
+	if [[ ! -s ${ryaVagrant}.jar ]] ; then
+		echo "Downloading"
+		download --output ${ryaVagrant}.jar  https://dl.dropbox.com/s/dgw63m66nubyy4z/${ryaVagrant}.jar?raw=1 || exit 120
+	fi
+	sudo mkdir --parents ${ryaVagrant}
+	sudo unzip -q -o ${ryaVagrant}.jar -d ${ryaVagrant}
+	sudo cp ${ryaVagrant}/*.xsl /var/lib/tomcat7/webapps/openrdf-workbench/transformations/
```

Comment:
In my install (where the rya files were downloaded from my website, not dropbox), tomcat user (tomcat7) had no permissions on the create-RyaAccumulSail.xsl file in the /var/lib/tomcat7/webapps/openrdf-workbench/transformations. I suggest adding 
sudo chown tomcat7:tomcat7 /var/lib/tomcat7/webapps/openrdf-workbench/transformations/*
after copying the *.xsl files